deploy/kubernetes/snapshot-controller/rbac-snapshot-controller.yaml

```diff
@@ -3,6 +3,7 @@ apiVersion: v1
 kind: ServiceAccount
 metadata:
   name: snapshot-controller
+  namespace: kube-system # TODO: replace with the namespace you want for your controller
```

Comment:
I just realized that we can't make this change here because this rbac file is actually used by the prow job.

Instead of changing the namespace to "kube-system", can you just comment it out and add a comment to explain why it makes sense to deploy it in a different namespace?

```
# namespace: kube-system # TODO: replace with the namespace you want for your controller
```

Reply:
ok - i take it prow cannot override the namespace in the yaml? Should I also update the USAGE.md to direct users to set the namespace to something here:

```
If your cluster does not come pre-installed with the correct components, you may manually install these components by executing the following steps.
```

Reply:
Prow will just use this rbac directly.

Sure, go ahead and add some comments in the Usage section to clarify about the namespace.

Reply:
Updated README and YAML files with more info about namespace and reverted the namespace to default for use by prow job.